Unable to install on AIX 7.2

Hello everyone,

my environment is:
OS: AIX 7.2
go version: go version go1.16.12 aix/ppc64

**Problem**:
When I try to install gotestsum with either of the suggested install methods `go install gotest.tools/gotestsum@latest` or `go get gotest.tools/gotestsum`, it results in the following errors:

```
$ go get -v gotest.tools/gotestsum 
[...]
gotest.tools/gotestsum/internal/filewatcher
# gotest.tools/gotestsum/internal/filewatcher
pkg/mod/gotest.tools/gotestsum@v1.8.0/internal/filewatcher/term_unix.go:43:40: undefined: tcGet
pkg/mod/gotest.tools/gotestsum@v1.8.0/internal/filewatcher/term_unix.go:50:38: undefined: tcSet
pkg/mod/gotest.tools/gotestsum@v1.8.0/internal/filewatcher/term_unix.go:58:37: undefined: tcSet
pkg/mod/gotest.tools/gotestsum@v1.8.0/internal/filewatcher/watch.go:31:18: undefined: fsnotify.NewWatcher
pkg/mod/gotest.tools/gotestsum@v1.8.0/internal/filewatcher/watch.go:99:25: undefined: fsnotify.Watcher
pkg/mod/gotest.tools/gotestsum@v1.8.0/internal/filewatcher/watch.go:197:32: undefined: fsnotify.Watcher
```

**Expected**: gotestsum is available in the defined user install directory.

Given that it is a similar missing symbol as in [this vipe issue](https://github.com/spf13/viper/issues/922) or [afero issue](https://github.com/spf13/afero/pull/210) I presume that AIX's implementation of the Filesystem is a bit different to how it works on other UNIX systems.

Not having found any other reference in the issues of gotestsum regarding AIX, I presume this hasn't come up yet. 
Is it possible to add/implement support for AIX? Happy to help there to provide any help/info where I can.

--- a/internal/filewatcher/term_aix.go
+++ b/internal/filewatcher/term_aix.go
@@ -1,0 +1,9 @@
+//go:build aix
+// +build aix
+
+package filewatcher
+
+import "golang.org/x/sys/unix"
+
+const tcGet = unix.TCGETA
+const tcSet = unix.TCSETA

--- a/internal/filewatcher/term_bsd.go
+++ b/internal/filewatcher/term_bsd.go
@@ -1,3 +1,4 @@
+//go:build darwin || dragonfly || freebsd || netbsd || openbsd
 // +build darwin dragonfly freebsd netbsd openbsd
 
 package filewatcher

--- a/internal/filewatcher/term_unix.go
+++ b/internal/filewatcher/term_unix.go
@@ -1,5 +1,5 @@
-//go:build !windows
-// +build !windows
+//go:build !windows && !aix
+// +build !windows,!aix
 
 package filewatcher
 

--- a/internal/filewatcher/watch.go
+++ b/internal/filewatcher/watch.go
@@ -1,3 +1,6 @@
+//go:build !aix
+// +build !aix
+
 package filewatcher
 
 import (

--- a/internal/filewatcher/watch_unsupported.go
+++ b/internal/filewatcher/watch_unsupported.go
@@ -1,0 +1,18 @@
+//go:build aix
+// +build aix
+
+package filewatcher
+
+import (
+	"fmt"
+	"runtime"
+)
+
+type Event struct {
+	PkgPath string
+	Debug   bool
+}
+
+func Watch(dirs []string, run func(Event) error) error {
+	return fmt.Errorf("file watching is not supported on %v/%v", runtime.GOOS, runtime.GOARCH)
+}

